### DISCUSSION TITLE
Layout question

### DISCUSSION THREAD

**Discussion by uwewiest** [Gulivertx/cbatmo#58] [General]
Hi Cedric,

first of all great work ! Found your project here and installed it on a spare 7" 1024x600 touch LCD and it looks great, had a pi3 at the beginning but way to slow and now with the pi4 ..just great.

Here is my question how can you influence which modules go where in the layout of the screen.?
I didn't find anything so maybe you can share some insights?

Especially when you want the weather forecast bigger etc

![grafik](https://user-images.githubusercontent.com/25363706/103381069-fd746b00-4aea-11eb-8cc5-e33f0a98e62c.png)

Thanks

Uwe

**Comment by Gulivertx** [comment]
Hi, thanks for your message and to use CBatmo! :)
Hummmm, layouts are not easy to modify, you have to touch typescript, SCSS and JSX code in multiple files. I cannot explain what you have to do because is too complicate:  But what I can tell you is you should have the knowledge about this 3 topics if you want to manage a new layout. Then you have to play with all components files and scss.

The problem of my design code to build the layout is that is for fixed screen pixels, and this is not good.
A modification of the complete layout should be done to handle multiple screen size with responsive design.

I also need to handle CBatmo to a different screen size, I need to think about a huge design modification to be able to handle multiple screen...

**Reply by uwewiest** [reply]
Hi yes I had a look and I can tell you have been working quite a while on this..and yes I know CSS etc but I would prefer to stick with your version :-) so if you will modify it so it would work with other screens e.g. 1024x600 would be great
